.github/pull_request_template.md

```diff
@@ -6,5 +6,11 @@
 ## Test Notes/Sample Code
 <!--- Notes about testing or code to reproduce new functionality --->
 
+
+## Connected Issues
+<!--- Links to issues, using "Closes #NNN" if the issue is closed via PR --->
+
+- Closes #XXX
+
 Notes: 
```

Comment:
This "Notes:" thing outside a header confuses me, so I also dropped that from the gsm.app template. Should it be its own header, or should it go away? Otherwise it feels like it's kinda sorta part of whatever is above it.

Reply:
to me it seems like `Test Notes/Sample Code` section probably covers it, so i'll drop it from here.